pkg/kubectl/cmd/cmd.go

```diff
@@ -390,21 +390,41 @@ func (h *DefaultPluginHandler) Execute(executablePath string, cmdArgs, environme
 	return syscall.Exec(executablePath, append([]string{executablePath}, cmdArgs...), environment)
 }
 
+// skipPrependedFlags iterates over all the arguments and return
+// the flags before plugin name and the remaining arguments
+func skipPrependedFlags(args []string) ([]string, []string) {
+	inFlag := false
+	for idx, arg := range args {
+		switch {
+		case strings.HasPrefix(arg, "-"):
+			if !strings.ContainsRune(arg, '=') {
```

Comment:
I think this is OK.

At first, I was thinking what about boolean flags that don't require an arg, like `--all-namespaces=true` vs. `--all-namespaces`.

But it turns out that even with built-in commands in the current version of kubectl, using `--all-namespaces` before the command does not work.

This works:
```
kubectl --all-namespaces=true get pods
```
But this does not...
```
kubectl --all-namespaces get pods
```
Even though this does...
```
kubectl get pods --all-namespaces
```

Reply:
yeah, cobra :wink: